doc/user-getting-started-c.md

```diff
@@ -41,7 +40,7 @@ Now we can create a CPIO named `rootfs` out of the folder `appdir` with:
 myst mkcpio appdir rootfs
 ```
 
-## Run the program inside a SGX enclave
+## Run the program inside an SGX enclave
```

Comment:
```suggestion
## Run the program inside a SGX enclave
```

Reply:
Hmmm, I started with "a SGX". @mingweishih suggested "an SGX". And I searched OE documentations, noticed that most of them use "an SGX".